### ISSUE TITLE
Support fat arrow returns

### ISSUE THREAD

**Issue by dcalhoun** [buxlabs/amd-to-es6#16]
It appears that there is no support for AMD blocks the utilize ES6 fat arrow implicit return. For example, the following block...
```javascript
define(['marionette'], Marionette => Marionette.ItemView.extend({}))
```

Throws the following error...
```
/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/src/lib/generateCode.js:99
    var nodes = code.filter(function (node) {
                    ^

TypeError: Cannot read property 'filter' of undefined
    at module.exports (/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/src/lib/generateCode.js:99:21)
    at Module.convert (/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/src/class/Module.js:24:20)
    at module.exports (/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/src/converter.js:14:12)
    at /Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/bin/cli.js:53:24
    at Array.forEach (native)
    at convertFiles (/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/bin/cli.js:50:11)
    at Object.<anonymous> (/Users/username/Sites/web/gn-web/node_modules/@buxlabs/amd-to-es6/bin/cli.js:71:5)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

Is there a way to support converting this syntax? Thanks!

**Comment by emilos** [comment]
hey @dcalhoun, thanks for the bug report! Fixed here https://github.com/buxlabs/amd-to-es6/commit/66cc1668bdc1b5f6fc2590f8435f68268116b852 and released as `0.8.1`

**Comment by dcalhoun** [comment]
Thank you _so much_! You just made my day with how quick you fixed this. 😄 🎉
